.circleci/config.yml

```diff
@@ -713,14 +741,19 @@ jobs:
           at: "."
       - packer_build:
           template: tools/packer/lotus-snap.pkr.hcl
-  publish-dockerhub:
+
+  build-dockerhub:
```

Comment:
Why change name?  The old name matches description better.

Reply:
I removed this for now though. It was a minor change, and not really that important.